couchpotato/static/style/combined.min.css

```diff
@@ -1,6 +1,6 @@
 .page.movie_details,.thumb_list .movie.hover_start .actions{pointer-events:none}
 .movies>.description a:hover,.page.movie_details .releases .buttons a:hover{text-decoration:underline}
-.update.message{background:#FFF;padding:20px;text-align:center;font-size:1.25em}
+.update.message{background:#fff;padding:20px;text-align:center;font-size:1.25em}
```

Comment:
And this file I didn't change too.. Here are just lowercasing (my `grunt` make all **FFF** small, and I don't know why)